src/components/molecules/Button/styles.module.scss

```diff
@@ -0,0 +1,104 @@
+@import 'src/styles/variables';
+@import 'src/styles/mixins';
+
+.baseButton {
+  display: flex;
+}
+
+.buttonText {
+  order: 2;
+}
+
+.m {
```

Comment:
**[blocker]**

Currently button sizes don't seem to be correct. At least the small button height is off. You are defining .m and .s styles here, but overwriting them with the .secondary or .primary classes, so any overlapping properties will just use whatever the classes for "appearance" dictate.